forms-flow-theme/scss/_modal.scss

```diff
@@ -213,4 +213,201 @@
   max-width: 100%;
   margin: auto;
   
-}
\ No newline at end of file
+}
+
+//settings modal
+
+.settingsModal{
+  display: flex;
+flex-direction: column;
+align-items: flex-start;
+align-self: stretch;
```

Comment:
Can we use Bootstrap classes for `display: flex;
flex-direction: column;
align-items: flex-start;
align-self: stretch;`? Also, since the border radius is the same for all modals, do we need to create a separate class for that?

Reply:
padding also